src/hm-admin/hm-admin.component.ts

```diff
@@ -1,22 +1,31 @@
 import {Component, Input, OnInit} from '@angular/core';
+import {HTTP_PROVIDERS} from '@angular/http';
 import {ConfigService} from './services/config/config.service';
+import {HydraCore} from './services/hydra/HydraCore.service';
 
 @Component({
   moduleId: module.id,
   selector: 'hm-admin-app',
   templateUrl: 'hm-admin.component.html',
   styleUrls: ['hm-admin.component.css'],
-  providers: [ConfigService]
+  providers: [ConfigService, HydraCore, HTTP_PROVIDERS]
 })
 export class HmAdminComponent implements OnInit {
   @Input('config')
   config: any;
-
-
-  constructor(private configService:ConfigService) {}
+  public hydraCoreDefinition;
+  constructor(private configService: ConfigService, private hydraCore: HydraCore) {}
 
   ngOnInit() {
+    let definition;
     this.configService.hookMeIfYouCan(this.config);
+    this.hydraCoreDefinition = this.hydraCore.getCacheHydraDefinition();
+
+    if (typeof this.hydraCoreDefinition === 'undefined') {
+      definition = this.hydraCore.getHydraCore('https://www.w3.org/ns/hydra/core');
```

Comment:
the url should be in the config file (maybe we don't want to use hydra or we want to use custom definitions)